activity-be/inoculation.sql

```diff
@@ -0,0 +1,17 @@
+create table inoculation
+(
+	id serial not null
+		constraint inoculation_pkey
+			primary key,
+	drug varchar(50) not null,
+	dose varchar(50) not null,
+	brand varchar(50),
+	product varchar(50),
+	description varchar(50),
+	application_date date default now(),
+	establishment_cuig varchar(50) not null,
```

Comment:
IDK the benefits of reducing this field to 10  characters, but we shouldn't have an establishment_cuig grater than 6 characters